README.md

```diff
@@ -0,0 +1,112 @@
+Elasticsearch JNA Build
+=======================
+
+This project exists in order to build a jna jar which supports
+platforms supported by elastic.  It builds native linux bits for jna
+in vagrant, and uses the provided native bits for mac and windows.
+
+
+Publish to Maven Central (Normal case)
+--------------------------------------
+
+In the default case, we track with upstream JNA.  This is when the
+local version exactly matches net.java.dev/jna.
+
+First set shell variables for convenience:
+
+    # match whatever's in build.gradle
+    export VERSION=4.4.0
+
+Ensure you have the OSSRH Sonatype staging environment set up in
+`~/.m2/settings.xml`:
+
+    <server>
+      <id>sonatype-nexus-staging</id>
+      <username>user</username>
+      <password>pass</password>
+    </server>
+
+Then build:
+
+    % vagrant up
+    % gradle assemble generatePomFileForJnaPublication
```

Comment:
Just `gradle assemble` should be sufficient here (assemble is set to depend on the pom generation task).